subgraph/tests/.docker/Dockerfile

```diff
@@ -1,21 +1,22 @@
-FROM --platform=linux/x86_64 ubuntu:20.04
+FROM --platform=linux/x86_64 ubuntu:22.04
```

Comment:
Not sure these changes belong in this PR?
If they are required, can you add them to _another_ PR because this is not related to the receipts directly.